Makefile

```
@@ -255,11 +255,15 @@ argoexec-image:
 	if [ $(DOCKER_PUSH) = true ] && [ $(IMAGE_NAMESPACE) != argoproj ] ; then docker push $(IMAGE_NAMESPACE)/$*:$(VERSION) ; fi
 
 .PHONY: codegen
-codegen: types swagger manifests $(GOPATH)/bin/mockery docs/fields.md docs/cli/argo.md
+codegen: types swagger manifests nix-hash $(GOPATH)/bin/mockery docs/fields.md docs/cli/argo.md
```

I think this makes more sense inside of the [`lint`](https://github.com/argoproj/argo-workflows/blob/42262690fcbab0ed6106d3f9069e128655b3339f/Makefile#L448) job, which already lints deps for both Go and Node/JS.

The script can be one-lined in there without a new `make` job as well

```suggestion
codegen: types swagger manifests $(GOPATH)/bin/mockery docs/fields.md docs/cli/argo.md
```